OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -94,10 +105,20 @@ public AssetBrowserLogic(Widget widget, Action onExit, ModData modData, WorldRen
 				sourceDropdown.GetText = () => sourceName.Update(assetSource);
 			}
 
+			var assetTypeDropdown = panel.GetOrNull<DropDownButtonWidget>("ASSET_TYPES_DROPDOWN");
+			if (assetTypeDropdown != null)
+			{
+				assetTypeDropdown.OnMouseDown = _ =>
+				{
+					assetTypeDropdown.RemovePanel();
+					assetTypeDropdown.AttachPanel(CreateAssetTypesPanel());
```

Comment:
Perhaps cache the call to `CreateAssetTypesPanel` in a var just outside the labmda?

Reply:
Done.